src/integrations/third-party/amp.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Integrations\Front_End_Integration;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * AMP integration
+ */
+class AMP implements Integration_Interface {
+
+	/**
+	 * The front end integration.
+	 *
+	 * @return Front_End_Integration
```

Comment:
I think this should be an `@var` 🙂. 
```suggestion
	 * @var Front_End_Integration
```

Reply:
Since this is the only remark that I have on the code I am going to fix it myself and let it go through the process.